packages/renderer/src/lib/preferences/PreferencesAuthenticationProvidersRendering.svelte

```diff
@@ -113,6 +119,23 @@ import SettingsPage from './SettingsPage.svelte';
                 {/each}
               </DropdownMenu>
             {/if}
+            <!-- Authentication Provider Auth Requests DropDown end -->
+
+            <!-- Authentication Provider Auth Request Sign In button start -->
+            {#if sessionRequests.length === 1}
```

Comment:
maybe this one should be first and then we replace the previous if by an else if ?